pkgs/servers/oxigraph/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, stdenv
+, rustPlatform
+, rustfmt
+, fetchFromGitHub
+, pkg-config
+, IOKit
+, Security
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "oxigraph";
+  version = "0.3.11";
+
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-7KbDZKKJPk3QTp4siIbdB6xKbslw73Lhc7NoeOuA0Og=";
+    fetchSubmodules = true;
+  };
+
+  cargoSha256 = "sha256-Yqn6hwejg6LzcqW0MiUN3tqrOql6cpu/5plaOz+2/ns=";
+
+  nativeBuildInputs = [
+    pkg-config
+    rustPlatform.bindgenHook
+  ] ++ lib.optional stdenv.isDarwin rustfmt;
+  buildInputs = lib.optionals stdenv.isDarwin [ IOKit Security ];
+
+  preConfigure = ''
+    cd server
+  '';
+  postBuild = ''
+    cd ..
+  '';
```

Comment:
does `cargoBuildFlags = [ "--package" "server" ]` achieve the same effect?

Reply:
```
error: package ID specification `server` did not match any packages
```

Reply:
Or maybe `oxigraph_server`?

Reply:
That did the trick!